junorunner/testrunner.py

```diff
@@ -1,11 +1,17 @@
-from __future__ import print_function
-
 from django.db import connections
+from django.conf import settings
 from django.core import management
 
 from .runner import JunoDiscoverRunner
 
 
+RERUN_LOG_FILE_NAME = getattr(
+    settings,
+    'TEST_RUNNER_RERUN_LOG_FILE_NAME',
+    'test_rerun.txt'
+)
```

Comment:
I don't see any writing to this file...

Reply:
There is no writing there is reading. Writing is in TextTestResult in extended_runner.py (the file already exists)